app/services/downstream_service.rb

```diff
@@ -18,6 +18,11 @@ def self.update_draft_content_store(downstream_payload)
       message = "Can only send draft, published and unpublished items to draft content store"
       raise DownstreamInvalidStateError.new(message)
     end
+    if downstream_payload.state != "draft" && draft_at_base_path?(downstream_payload.base_path)
+      message = "Can't send #{downstream_payload.state} item to draft content store, as there is a draft occupying the same base path"
+      raise DownstreamInvalidStateError.new(message)
```

Comment:
What do you mean by "error situation"? Disregarding some bug in submitting the job, this particular condition just means that the world state has changed between the job being submitted, and being given to a worker, which is perfectly normal, and not something that I consider an error.